phobos/io/parser.py

```diff
@@ -2,13 +2,15 @@
 from xml.etree import ElementTree as ET
 
 from .xmlrobot import XMLRobot
+from ..utils.commandline_logging import setup_logger_level, get_logger
+log = get_logger(__name__)
```

Comment:
Warum ist hier get_logger und nicht setup_logger_level. und warum ist setup_logger_level in Robot

Reply:
Weil das oben in der Tat falsch ist. Man muss in jedem File nur pro Modul quasi einmal einen Logger bestellen mit get_logger(__name__)